File: ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureLauncherCallback.java
package com.braintreepayments.api;

import androidx.annotation.NonNull;

/**
 * Callback for receiving the results via {@link ThreeDSecureLauncher}
 */
public interface ThreeDSecureLauncherCallback {

    void onThreeDSecurePaymentAuthResult(@NonNull ThreeDSecurePaymentAuthResult paymentAuthResult);
}

File: ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureLauncherCallback.kt
package com.braintreepayments.api

/**
 * Callback for receiving the results via [ThreeDSecureLauncher]
 */
fun interface ThreeDSecureLauncherCallback {

    fun onThreeDSecurePaymentAuthResult(paymentAuthResult: ThreeDSecurePaymentAuthResult)
}
